addons/website_rating_project/static/src/css/website_rating_project.css

```diff
@@ -1,3 +1,32 @@
 .oe_website_rating_project .thumbnail{
-    height: 355px;
+    height: 240px;
 }
+.o_top_partner_rating_image {
+    height: 15px;
+}
+.o_top_partner_image {
+    height: 30px;
+    width: 30px;
+}
+.o_top_partner_feedback{
+    word-wrap: break-word;
+}
+.o_vertical_separator {
+    border-left: 1px solid #eeeeee
+}
+.o_rating_progress {
+    margin-bottom: 10px;
+}
+.o_rating_count {
+    display: inline-block;
+    min-width: 22px
+}
+.o_smiley_no_padding_left {
+    padding-left: 0;
+}
+.o_smiley_no_padding_right {
+    padding-right: 0;
+}
+.o_lighter_smileys {
+    opacity: 0.4
+}
```

Comment:
Maybe it's time to convert this file into a LESS one. Make sure to 'namespace' all this classes to avoid collition.
```
.o_website_rating_project_container {
 .o_my_class1{ .... }
 .o_my_class2{ .... }
}
``